File: Northwind_Demo/Sources/NorthWind.Web.Angular/src/app/domain/serverActionServices/reportService.js
import CodeBehindReportService from './codeBehind/codeBehindReportService';

export default class ReportService extends CodeBehindReportService{
	constructor(
		//--  custom dependencies
		//-- /custom dependencies
		customActionRunner) {
		"ngInject";	
		super(customActionRunner);
	}
}

File: Northwind_Demo/.origin/Sources/NorthWind.Client.ServerServices/ActionServices/Server/ReportService.cs
using System;
using BusinessFramework.Client.Contracts.Connection;
using BusinessFramework.Contracts.Actions;


namespace NorthWind.Client.ServerServices.ActionServices.Server
{

    /// <summary>
    /// 
    /// </summary>
    public sealed class ReportService : CodeBehind.CodeBehindReportService
    {    

        /// <summary>
        /// Ctor
        /// </summary>
        public ReportService(
            //--  custom dependencies
            //-- /custom dependencies
            IControllerClientFactory clientFactory)
            : base(clientFactory)
        {
        }
    }
}